src/lualib/5.0/Generator.ts

```diff
@@ -0,0 +1,28 @@
+import { __TS__CountVarargs } from "./CountVarargs";
```

Comment:
Again don't really like the code duplication. But also no real idea how to work around it, except refactoring into a common base, which might be even messier. Would be cool if we could come up with something, otherwise I am also fine with keeping it this way.

Reply:
With the help of the unpack helper, it turns out we don't have to duplicate this file at all! https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/9820e42473c8b8617d62a1a5887fe4a54359423c